phpunit/class-wp-theme-json-test.php

```diff
@@ -899,10 +884,6 @@ public function test_merge_incoming_data_null_presets() {
 						),
 					),
 				),
-				'layout'     => array(
-					'contentSize' => '610px',
```

Comment:
not sure what this test is about but this config is valid (contentSize) and maybe we could keep it.

Reply:
This commit is removing a previous commit in this PR: I had added this when I thought `layout.units` was a valid input and wanted to make sure the merge algorithm respected pre-existing data (`layout.contentSize`). If we want a test for this it should live in another method of this file. Can be done in a follow-up PR if necessary.